virttest/utils_stress.py

```diff
@@ -158,3 +163,27 @@ def vm_stress_events(self, event, vm):
                         libvirt.delete_local_disk(self.disk_type, disk_name)
             else:
                 raise NotImplementedError
+
+    def host_stress_event(self, event):
+        """
+        Host Stress events
+
+        :param event: event name
+        """
+        for itr in range(self.host_iterations):
+            if "cpufreqgovernor" in event:
+                cpu.set_cpufreq_governor()
+                logging.debug("Current governor: %s", cpu.get_cpufreq_governor())
+                time.sleep(self.event_sleep_time)
+            elif "cpuidlestate" in event:
+                idlestate = cpu.get_cpuidle_state()
+                cpu.set_cpuidle_state()
+                time.sleep(self.event_sleep_time)
+                cpu.set_cpuidle_state(setstate=idlestate)
```

Comment:
how do we make sure the cpu idle state is set or not ? can we do get_cpuidle_state() after setting to validate ?

Reply:
validation will make the stress events heavier, there is check in the `set_cpuidle_state` function itself and that would suffice.